build.sbt

```diff
@@ -22,8 +22,8 @@ developers := List(
   )
 )
 
-scalaVersion := "2.13.2"
-crossScalaVersions := Seq("2.11.12", "2.12.10", "2.12.11", "2.13.1", "2.13.2")
+scalaVersion := "2.13.3"
+crossScalaVersions := Seq("2.11.12", "2.12.10", "2.12.11", "2.13.1", "2.13.2", "2.13.3")
```

Comment:
Should we drop `2.13.1` then?

Reply:
not sure about that... is it possible anyone is stuck on 2.13.1 ? Probably not.

Reply:
Yeah, we agreed to publish only for 2 latest versions of 2.12 and 2.13, so I'd also be in favour of removing 2.13.1 before the next release.

Reply:
https://github.com/sksamuel/scapegoat/pull/401